src/sinks/pulsar.rs

```diff
@@ -225,6 +227,27 @@ mod tests {
 
         assert_eq!(&event[..], msg.as_bytes());
     }
+
+    #[test]
+    fn pulsar_encode_event() {
+        let msg = "hello_world";
+
+        let mut evt = Event::from(msg);
+        evt.as_mut_log().insert("key", "value");
+
+        let event = encode_event(
+            evt,
+            &EncodingConfigWithDefault {
+                codec: Encoding::Text,
+                except_fields: Some(vec![Atom::from("key")]),
```

Comment:
I'm not sure what this is testing. Encoding to text doesn't usually include fields other than `message`, so I feel like we shouldn't even allow `except_fields` in this case.

Reply:
This is not correct test, thanks! I'll change to `Encoding::Json`.